packages/nns/src/canisters/governance/response.converters.ts

```diff
@@ -680,10 +698,32 @@ const convertPbPrincipalIdToPrincipalString = (
 ): string =>
   Principal.fromUint8Array(pbPrincipal.getSerializedId_asU8()).toText();
 
-const convertPbNeuronToFullNeuron = (
-  pbNeuron: PbNeuron,
-  pbNeuronInfo: PbNeuronInfo
-): Neuron => {
+const convertNeuronSubaccountToAccountIdentifier = ({
+  neuron,
+  canisterId,
+}: {
+  neuron: PbNeuron;
+  canisterId: Principal;
+}): AccountIdentifier => {
+  const subAccount = SubAccount.fromBytes(
```

Comment:
what happens in case `SubAccount` returns an `Error`?
if it can really be safely ignored, maybe a comment?

Reply:
Good idea, I will add a comment.